docs/src/main/paradox/docs/delta/api/index.md

```diff
@@ -24,6 +24,7 @@
 * @ref:[Jira integration](jira.md)
 * @ref:[Events](events-api.md)
 * @ref:[Supervision](supervision-api.md)
+* @ref:[Id Resolution](id-resolution.md)
```

Comment:
Maybe put it before or after multi-fetch ?
It seems way more useful than the last ones